### PR TITLE
Darkmode incongruencies

### LINKED ISSUE #137: [UI/UX] Maintain dark mode for website
Currently there are a few parts of the website that are not compatible with dark mode such as:

Pop-up updates message
Field preset

### PR DESCRIPTION
**Pull request by Thine-Sho**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -92,7 +92,6 @@ <h5 class="modal-title">April 26, 2021 Updates</h5>
             <div class="navbar-nav">
                 <a class="nav-item nav-link" href="https://github.com/pioneers/runtime/wiki/Student-API" target="_blank">Robot API</a>
                 <a class="nav-item nav-link" href="https://github.com/pioneers/pimulator-js/blob/master/releasenotes.md" target="_blank">Release Notes</a>
-                <a class="nav-item nav-link" href="https://forms.gle/i4q48q7fxBxRxDSb6" target="_blank">Bug Reports/Feature Requests</a>
```
Comment (kevinzhu00) [inline templates/index.html]: Make sure to only commit changes that are relevant to your code. It seems that your branch is behind. Make sure to pull before committing.

File: templates/index.html [hunk 2]
```diff
@@ -767,6 +766,21 @@ <h3> <b>Line Follower</b> </h3>
                                                 </div>
                                             </div>
                                         </div>
+
```
Comment (kevinzhu00) [inline templates/index.html]: Same as prior comment.

**Review by kevinzhu00 — DISMISSED** [review]

File: templates/index.html [hunk 3]
```diff
@@ -767,6 +767,21 @@ <h3> <b>Line Follower</b> </h3>
...
+                                                    <summary>Feedback/Bug Reports</summary>
+                                                </details>
+                                            </div>
+                                            <div id="help-debug" class="collapse" aria-labelledby="help-debug">
+                                                <div class="card-body">
+                                                    If you encountered a bug that you feel requires attention from staff, please submit a report at this link:
+                                                    <a href="https://forms.gle/i4q48q7fxBxRxDSb6" target="_blank"> Submit a Bug Report </a>
+                                                </div>
+                                            </div>
+                                        </div>
```
Comment (charleshong3) [inline templates/index.html]: It looks like this was also added in by accident while resolving a merge conflict. It should be deleted.

**Review by charleshong3 — CHANGES_REQUESTED** [review]

**Review by nia-n — APPROVED** [review]
lgtm

**Review by charleshong3 — APPROVED** [review]

**Pull request merged**